### PR TITLE
Support multiple signals for a single handler

### PR DESCRIPTION
**Pull request by sethrj**
This should allow us to gracefully die when Celeritas is killed by a job scheduler. I've also fixed the build flags for ndebug-novg on summit (oops!)

### CODE REVIEW

File: src/comm/ScopedSignalHandler.cc [hunk 1]
```diff
@@ -71,13 +72,37 @@ bool ScopedSignalHandler::allow_signals()
     return result;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Whether signal handling is enabled.
```
Comment (amandalund) [inline src/comm/ScopedSignalHandler.cc:77]: copy-paste

**Review by amandalund — APPROVED** [review]
Looks good :)

**Comment by sethrj** [comment]
@amandalund I'm piggy-backing the summit fixes (and the addition of OpenMP to the config output) onto this MR... 😬

**Comment by amandalund** [comment]
No prob, I was expecting that 🙂

**Pull request merged**